recipes/pyramid_log/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "pyramid_log" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # If getting the source from GitHub, remove the line above,
+  # uncomment the line below, and modify as needed. Use releases if available:
+  # url: https://github.com/simplejson/simplejson/releases/download/{{ version }}/simplejson-{{ version }}.tar.gz
+  sha256: 165ce9460548b09061db6023b5079eb6290d9490a2b4cf28946e3515185d0a52
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    # - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pyramid
+
+test:
+  # Some packages might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+    - pyramid_log
+
+about:
+  home: https://pypi.org/project/pyramid-log/
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'Python logging formatter for Pyramid'
+
```

Comment:
Please remove all comments and empty line here.

Reply:
Cool -- nice to know, I've been having a lot of pain with Appveyor!

Stay tuned for a cleaned up meta.yaml